stable/jenkins/templates/config.yaml

```diff
@@ -221,15 +221,20 @@ data:
     cp /var/jenkins_config/config.xml /var/jenkins_home;
     cp /var/jenkins_config/jenkins.CLI.xml /var/jenkins_home;
     cp /var/jenkins_config/jenkins.model.JenkinsLocationConfiguration.xml /var/jenkins_home;
-  {{- else }}
+{{- if .Values.master.additionalConfig }}
```

Comment:
I suggest to use indenation as it makes it clearer where a block starts and ends.